Dockerfile

```diff
@@ -2,6 +2,16 @@
 # Build the manager binary
 FROM golang:1.18 as builder
 
+ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
+
+RUN printf "Building for TARGETPLATFORM=${TARGETPLATFORM}" \
```

Comment:
So I ran this locally and I got the following output:

```
$ make container
docker buildx build \
	-f Dockerfile \
	--target distroless \
	--build-arg TAG=v2.3.1-74-gc3279dd4 \
	--build-arg COMMIT=c3279dd4 \
	--build-arg REPO_INFO=git@github.com:Kong/kubernetes-ingress-controller.git \
	-t kong/kubernetes-ingress-controller:v2.3.1-74-gc3279dd4 .
[+] Building 61.2s (16/17)
 => [internal] load build definition from Dockerfile                                                                                                                                           0.0s
 => => transferring dockerfile: 6.40kB                                                                                                                                                         0.0s
 => [internal] load .dockerignore                                                                                                                                                              0.0s
 => => transferring context: 35B                                                                                                                                                               0.0s
 => [internal] load metadata for docker.io/library/golang:1.18                                                                                                                                 0.0s
 => [internal] load metadata for gcr.io/distroless/static:nonroot                                                                                                                              0.0s
 => CACHED [distroless 1/3] FROM gcr.io/distroless/static:nonroot                                                                                                                              0.0s
 => CACHED [builder 1/9] FROM docker.io/library/golang:1.18                                                                                                                                   61.2s
 => [internal] load build context                                                                                                                                                              0.0s
 => => transferring context: 551.01kB                                                                                                                                                          0.0s
 => [builder 2/9] RUN printf "Building for TARGETPLATFORM=linux/amd64"     && printf ", TARGETARCH=amd64"     && printf ", TARGETOS=linux"     && printf ", TARGETVARIANT=${TARGETVARIANT} \n  0.2s
 => [builder 3/9] WORKDIR /workspace                                                                                                                                                           0.0s
 => [builder 4/9] COPY go.mod go.mod                                                                                                                                                           0.0s
 => [builder 5/9] COPY go.sum go.sum                                                                                                                                                           0.0s
 => [builder 6/9] RUN go mod download                                                                                                                                                         24.7s
 => [builder 7/9] COPY pkg/ pkg/                                                                                                                                                               0.0s
 => [builder 8/9] COPY internal/ internal/                                                                                                                                                     0.0s
 => [builder 9/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH="amd64" GO111MODULE=on go build -a -o manager -ldflags "-s -w -X github.com/kong/kubernetes-ingress-controller/v2/internal/manager/met  35.2s
 => [distroless 2/3] COPY --from=builder /workspace/manager .                                                                                                                                  0.1s
 => exporting to image                                                                                                                                                                         0.8s
 => => exporting layers                                                                                                                                                                        0.8s
 => => writing image sha256:8dddbffc638b2e0cdd8c9e167f8c1c74248fbf8f36d4d152ee4196f85137a314                                                                                                   0.0s
 => => naming to docker.io/kong/kubernetes-ingress-controller:v2.3.1-74-gc3279dd4
```

I don't think this was the intended output? Maybe there's another way we can do this?

Reply:
Yeah... for the output type of `tty` (which is what you'll get by default when running in terminal) you'll see exactly that. I "tested" it with `--progress plain` - which is what we should see on CI logs.

Reply:
I wont consider it a blocker as it's ultimately negligible, but if there's an easy improvement to make it look better for people building using the Makefile we should do so.

Reply:
But then it doesn't "pretty print" in the terminal. I'd leave it as is, observe what we get in CI logs and then decide if we'd like to do something with it.

Reply:
That's fair, and that's why I didn't consider this a blocker: if there's nothing quick we can do to have both outputs look good, it's a tiny enough thing we shouldn't spend time on it right now.